games/devtest/mods/testnodes/performance_test_nodes.lua

```diff
@@ -0,0 +1,59 @@
+-- Performance test mesh nodes
+
+local S = minetest.get_translator("testnodes")
+
+-- Complex mesh
+minetest.register_node("testnodes:performance_mesh_clip", {
+	description = S("Performance Test Node\nMarble with 'clip' transparency"),
```

Comment:
Descriptions should be translated like so: https://github.com/minetest/minetest/blob/cded6a3945206e51c35adbd063f1aec3a47e310f/games/devtest/mods/testnodes/param2.lua#L6-L7

Reply:
Concatenating inside sentences is bad, but it's a good idea to concatenate multiple senstences together.

To rephrase, here's how the modding book puts it:

![image](https://github.com/minetest/minetest/assets/2122943/beee2a56-8579-4188-b7e2-088c29858087)

Reply:
I can probably answer it myself: There is not any example in devtest of translated line breaks, according to `grep -P 'S\("[^"]*\\n'`.
I changed it as requested.